src/v2/view-builder/ViewFactory.js

```diff
@@ -120,6 +121,7 @@ const VIEWS_MAPPING = {
     'security_key': ChallengeWebauthnView,
     'security_question': ChallengeAuthenticatorSecurityQuestion,
     phone: ChallengeAuthenticatorPhoneView,
+    app: UserVerificationDeviceChallengePollView,
```

Comment:
- how about rename to `ChallengeAuthenticatorOktaVerifyView` ?
- curious the diff between `DeviceChallengePollView` and `UserVerificationDeviceChallengePollView`.  the implementation looks quite similar. what are use cases or we'll deprecate former?